jenkins.py

```diff
@@ -99,17 +107,14 @@ def _api_call(url, type, opener, http_timeout):
         urllib2.install_opener(opener)
         resp = urllib2.urlopen(url, timeout=http_timeout)
     except (urllib2.HTTPError, urllib2.URLError) as e:
-        collectd.error("Error making API call (%s) %s" % (e, url))
-        return None
-    try:
-        return json.load(resp)
-    except ValueError, e:
         if e.code == 500 and type == "healthcheck":
-            return json.load(resp)
+            return load_json(e, url)
```

Comment:
Did you mean to pass the exception to `load_json()`?

Reply:
Yes. Jenkins has a weird way of sending healthcheck issues as error 500. But we still need to send which of the sub part is unhealthy